model/infinispan/src/main/java/org/keycloak/models/cache/infinispan/RealmCacheSession.java

```diff
@@ -1024,8 +1024,14 @@ public Stream<GroupModel> getTopLevelGroupsStream(RealmModel realm, Integer firs
     }
 
     @Override
+    @Deprecated
     public Stream<GroupModel> searchForGroupByNameStream(RealmModel realm, String search, Integer first, Integer max) {
-        return getGroupDelegate().searchForGroupByNameStream(realm, search, first, max);
+        return searchForGroupByNameStream(realm, search, false, first, max);
```

Comment:
This can be moved to the default method in the `GroupProvider` interface.

Reply:
I agree with this suggestion, please fix.

Reply:
I want to verify a couple of details here. I still need to have an implementation of the new, non-deprecated method which is being changed here. Is the suggestion to provide an implementation of both? 

I took the suggestions here the first time around and implemented it as the default for `GroupLookupProvider` instead, is this not adequate?

Reply:
> I took the suggestions here the first time around and implemented it as the default for `GroupLookupProvider` instead, is this not adequate?

This is exactly where it belongs. Thanks